File: startup-rest-web-service-archetype/src/main/resources/archetype-resources/__rootArtifactId__-domain/src/main/java/dm/__moduleName__/dto/__entityName__DTO.java
#set( $symbol_pound = '#' )
#set( $symbol_dollar = '$' )
#set( $symbol_escape = '\' )
package ${package}.dm.${moduleName}.dto;

import java.util.Date;

import org.startupframework.dto.EntityDTO;

import com.fasterxml.jackson.annotation.JsonFormat;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class ${entityName}DTO extends EntityDTO {
	
	public ${entityName}DTO() {		
	}
	
	private Integer numberField;
	private String stringField;
	
	@JsonFormat(pattern = DATE_PATTERN)
	private Date dateField;

}

File: startup-k-rest-service-archetype/src/main/resources/archetype-resources/__rootArtifactId__-domain/src/main/kotlin/dm/__moduleName__/dto/__entityName__DTO.kt
#set( $symbol_pound = '#' )
#set( $symbol_dollar = '$' )
#set( $symbol_escape = '\' )
package ${package}.dm.${moduleName}.dto

import java.util.Date
import org.startupframework.dto.EntityDTO
import com.fasterxml.jackson.annotation.JsonFormat

class ${entityName}DTO : EntityDTO() {

	var numberField: Int? = null
	var stringField: String? = null

	@JsonFormat(pattern = DATE_PATTERN)
	var dateField: Date? = null

}
